### PR TITLE
Feature/prepare be for city

### PR DESCRIPTION
**Pull request by blattlause**
All methods ready for city

### CODE REVIEW

File: LX.StaffScheduler.Api/Controllers/CitiesController.cs [hunk 1]
```diff
@@ -1,37 +1,98 @@
...
+                var existingCity = await _svc.GetByIdAsync(id);
+                if (existingCity == null)
+                {
+                    return NotFound("City not found");
+                }
+
+                existingCity.Name = cityDTO.Name;
+
+                await _svc.UpdateAsync(existingCity);
+                return NoContent();
```
Comment (IvanLukyanau) [inline LX.StaffScheduler.Api/Controllers/CitiesController.cs:70]: return updated object

File: LX.StaffScheduler.BLL/DTO/DistrictDTO.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{
+    public class DistrictDTO : IDTO
+    {
+        public int Id { get; set; }
+
+        public string DistrictName { get; set; }
+
+        public int CityId { get; set; }
+
+        //public CityDTO City { get; set; }
```
Comment (IvanLukyanau) [inline LX.StaffScheduler.BLL/DTO/DistrictDTO.cs:17]: remove it if not necessary

File: LX.StaffScheduler.BLL/DependencyInjection/MappingHelper.cs [hunk 3]
```diff
@@ -38,5 +33,37 @@ public static IEnumerable<CityDTO> ToDTO(this IEnumerable<City> City)
...
         }
 
+        public static District FromDTO(this DistrictDTO districtDTO)
+        {
+            return new District
+            {
+                Id = districtDTO.Id,
+                Name = districtDTO.DistrictName,
+                CityId = districtDTO.CityId,
+                //City = districtDTO.City.FromDTO()
```
Comment (IvanLukyanau) [inline LX.StaffScheduler.BLL/DependencyInjection/MappingHelper.cs:43]: remove commented string

File: LX.StaffScheduler.BLL/DependencyInjection/MappingHelper.cs [hunk 4]
```diff
@@ -38,5 +33,37 @@ public static IEnumerable<CityDTO> ToDTO(this IEnumerable<City> City)
...
+        }
+
+        public static DistrictDTO ToDTO(this District district)
+        {
+            return new DistrictDTO
+            {
+                Id = district.Id,
+                DistrictName = district.Name,
+                CityId = district.CityId,
+                //City = district.City.ToDTO()
```
Comment (IvanLukyanau) [inline LX.StaffScheduler.BLL/DependencyInjection/MappingHelper.cs:54]: remove comment

**Review by IvanLukyanau — CHANGES_REQUESTED** [review]

**Review by Karisha04 — APPROVED** [review]

**Pull request merged**